ceremonies/src/lib.rs

```diff
@@ -1248,33 +1248,33 @@ impl<T: Config> Pallet<T> {
 
 		<BootstrapperRegistry<T>>::remove_prefix(cc, None);
 		<BootstrapperIndex<T>>::remove_prefix(cc, None);
-		<BootstrapperCount<T>>::insert(cc, 0);
+		<BootstrapperCount<T>>::remove(cc);
 
```

Comment:
Actually, remove the storage keys instead of setting them to the default value. The implementation from before leaves us unfortunately some storage residues...